content/community/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Community"
+weight: 100
+---
+
+# Contributing to Paketo Buildpacks
+
+## How can I contribute to the project?
+
+Welcome! We're glad you have an interest in improving Paketo Buildpacks.
+Firstly, check out our [Contribution
+Guide](https://github.com/paketo-buildpacks/.github/blob/main/CONTRIBUTING.md)
+for an idea of where to begin. We also have a collection of [good first
+issues](https://github.com/search?q=org%3Apaketo-buildpacks+org%3Apaketo-community+label%3A%22good+first+issue%22+state%3Aopen&type=Issues)
+on our repositories if you're eager to start contributing.
+
+### RFCs
+
+If you'd like to learn more about the buildpacks architecture and the design
+decisions being made, it may be helpful to read through some of our
+[project-wide RFCs](https://github.com/paketo-buildpacks/rfcs). Some language
+family buildpack repositories (e.g.
+[Node.js](https://github.com/paketo-buildpacks/nodejs/tree/main/rfcs)) also
+contain language-specific RFCs that may provide helpful context.
```

Comment:
Could we add a link here to **all** open project-wide and language family RFCS? I think it's [this](https://github.com/search?q=org%3Apaketo-buildpacks+label%3Arfc&state=open&type=Issues), but we should double check.